### DISCUSSION TITLE
[Wh] driving motive behind or sustainability/funding model

### DISCUSSION THREAD

**Discussion by goyalyashpal** [flet-dev/flet#1669] [General]
title: Curious about driving motive behind or sustainability/funding model of flet
_[Wh]_ here means Wh-family word, like who, what, why, how, etc..

Hi!

as the title says, what is the motive behind developing this flet?
like what is it that you are gaining from developing this? 
_how_ and _why_ is it getting funded?

sorry, this may sound very absurd, but this was making me very curious...

**Comment by FeodorFitsner** [comment]
Flet is funded by AppVeyor - the company I co-founded and currently working at.
It's exciting to work on Flet! It's exciting to do open-source, interact with community, program for multiple platforms, build mobile apps! The motivation is to make creation of multi-platform apps easy for Python developers.
Going forward we will try turning Flet project into a sustainable business by offering paid products and services around open-source project.

Hope that helps. Let me know if you have any questions.

**Reply by goyalyashpal** [reply]
appveyor is used a lot in CI builds if i recall it right. wow, super nice. 

thanks a lot.
